__mocks__/axios.js

```diff
@@ -31,11 +32,13 @@ const urlDirectoryMap = {
   "mktorest.com": "marketo",
   "active.campaigns.rudder.com": "active_campaigns",
   "api.aptrinsic.com": "gainsight_px",
-  "api.profitwell.com": "profitwell"
+  "api.profitwell.com": "profitwell",
+  "ruddertest2.mautic.net":"mautic"
```

Comment:
This one too

Reply:
This one is used to get the directory for Response.json file. 
```


```